packages/ui-toolkit/src/form/toggle.js

```diff
@@ -73,24 +77,49 @@ const InputContainer = styled.div`
 const Li = BaseInput(styled.li`
   display: inline-block;
   list-style-type: none;
-  vertical-align: text-bottom;
-  border-top: ${border.unchecked};
-  border-bottom: ${border.unchecked};
-  border-left: ${border.unchecked};
-  &:first-of-type {
-    border-radius: ${borderRadius} 0 0 ${borderRadius};
-  }
-  &:last-of-type {
-    border-radius: 0 ${borderRadius} ${borderRadius} 0;
-    border-right: ${border.unchecked};
-  }
+  box-sizing: border-box;
 `);
 
+const Background = styled.span`
+  border-right: ${border.unchecked};
+  background-color: ${props => props.theme.white};
+  display: block;
+  border-radius: ${borderRadius};
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 0;
+  transition: all 200ms ease-out;
+  border: 0.0625rem solid #d8d8d8;
+  top: -1px;
+`;
+
 const Ul = styled.ul`
   display: inline-block;
   margin: 0;
   padding: 0;
-  box-shadow: ${bottomShaddow};
+  height: 48px;
+  display: inline-flex;
+  align-items: flex-end;
+  background-color: ${props => props.theme.disabled};
+  border-radius: ${borderRadius};
+  border: ${border.unchecked};
+  position: relative;
+  margin-top: ${unitcalc(1)};
+
+  li:first-of-type {
+    input + label + .background {
+      transform: translateX(calc(100% - 3px));
+    }
+
+    input:checked + label + .background {
+      transform: translateX(-1px);
```

Comment:
rem?

Reply:
I like how you don't condemn my hack but tell me to use rem 😝

Reply:
I'm all for clever hacks 😄